src/blockchain/schema.rs

```diff
@@ -16,93 +16,92 @@
     pub fn anchoring_transactions_chain(&self) -> RefMut<ProofListIndex<T, Transaction>> {
-        self.access.get_object(TRANSACTIONS_CHAIN)
+        self.access
+            .get_object(self.index_name("transactions_chain"))
     }
 
-    /// Returns the table that contains already spent funding transactions.
-    pub fn spent_funding_transactions(&self) -> RefMut<ProofMapIndex<T, Hash, Transaction>> {
-        self.access.get_object(SPENT_FUNDING_TRANSACTIONS)
+    /// Returns a table that contains already spent funding transactions.
+    pub fn spent_funding_transactions(&self) -> RefMut<ProofMapIndex<T, Sha256d, Transaction>> {
+        self.access
+            .get_object(self.index_name("spent_funding_transactions"))
     }
 
-    /// Returns the table that contains signatures for the given transaction input.
+    /// Returns a table that contains signatures for the given transaction input.
     pub fn transaction_signatures(&self) -> RefMut<ProofMapIndex<T, TxInputId, InputSignatures>> {
-        self.access.get_object(TRANSACTION_SIGNATURES)
+        self.access
+            .get_object(self.index_name("transaction_signatures"))
+    }
+
+    /// Returns an actual anchoring configuration entry.
+    pub fn actual_config_entry(&self) -> RefMut<Entry<T, Config>> {
+        self.access.get_object(self.index_name("actual_config"))
+    }
+
+    /// Returns a following anchoring configuration entry.
+    pub fn following_config_entry(&self) -> RefMut<Entry<T, Config>> {
+        self.access.get_object(self.index_name("following_config"))
     }
 
-    /// Returns a list of hashes of Exonum blocks headers.
-    pub fn anchored_blocks(&self) -> RefMut<ProofListIndex<T, Hash>> {
-        self.access.get_object(ANCHORED_BLOCKS)
+    /// Returns an entry that may contain an unspent funding transaction for the
+    /// actual configuration.
+    pub fn unspent_funding_transaction_entry(&self) -> RefMut<Entry<T, Transaction>> {
+        self.access
+            .get_object(self.index_name("unspent_funding_transaction"))
     }
 
-    /// Returns hashes of the stored tables.
+    /// Returns object hashes of the stored tables.
     pub fn state_hash(&self) -> Vec<Hash> {
         vec![
             self.anchoring_transactions_chain().object_hash(),
             self.spent_funding_transactions().object_hash(),
             self.transaction_signatures().object_hash(),
-            self.anchored_blocks().object_hash(),
         ]
     }
 
-    /// Returns the actual anchoring configuration.
-    pub fn actual_configuration(&self) -> GlobalConfig {
-        let actual_configuration = Schema::new(self.access.clone()).actual_configuration();
-        Self::parse_config(&actual_configuration)
-            .expect("Actual BTC anchoring configuration is absent")
+    /// Returns an actual anchoring configuration.
+    pub fn actual_configuration(&self) -> Config {
+        self.actual_config_entry().get().unwrap()
```

Comment:
Even if this entry is always set, I think it'd be better to have `expect` with a helpful description here.